agent/agent_test.go

```diff
@@ -484,7 +487,57 @@ post:
 		"Running command 'shell.exec' (step 1 of 1) in block 'post'",
 		"Finished command 'shell.exec' (step 1 of 1) in block 'post'",
 		"Finished running post-task commands",
-	}, []string{panicLog})
+	}, []string{
+		panicLog,
+		"Set idle timeout for 'shell.exec'",
+	})
+}
+
+func (s *AgentSuite) TestMainTask() {
```

Comment:
can we maybe either make the task name more descriptive or add a comment explaining what it is that we are asserting with this test?

Reply:
Done, also fixed the other ones already in this file with the same issue.